### DISCUSSION TITLE
Connect Waterrower with Smartrow to the App for Peloton Row

### DISCUSSION THREAD

**Discussion by Crazy-Ace** [cagnulein/qdomyos-zwift#1569] [General]
Hi Roberto,
I have had a WaterRower for a while now and recently purchased the Smart Row kit as well and was hoping to be able to connect it to my iOS qdomyos-zwift app so I could use the Peloton Row courses. Unfortunately, the rower did not connect to the app through Smart Row. Is there any way to connect it and use the courses?
I was so looking forward to using the Peloton Row courses with my rower and now that the courses are unlocked for Peloton App users, it would be awesome to be able to connect my rower as well.
Is there a feature for this in the domyos-zwift app, or is something like this planned in the future? Is there a beta Version of the iOS App I could test via TestFlight?
Thanks a lot in advance.

**Comment by cagnulein** [comment]
hi @Crazy-Ace the support for this rower is in the beta. 
send me an email to roberto.viola83 at gmail.com indicating this ticket and I will send you it
Let me know!

**Reply by Crazy-Ace** [reply]
Thanks Mate!
